test/unit/sth_test.js

```diff
@@ -61,7 +61,7 @@ describe('sth tests', function() {
     describe('server start', function() {
         it('should start gracefully', function(done) {
             sth.sthServer.startServer(sthConfig.STH_HOST, sthConfig.STH_PORT, function(err, server) {
-                expect(err).to.equal(null);
+                //expect(err).to.equal(null);
```

Comment:
Should be re-enabled?

Reply:
done in https://github.com/telefonicaid/fiware-sth-comet/pull/568/commits/183b07c461e0ae241d1005b3d14b12957b96d8e2